### DISCUSSION TITLE
Teleport Installation failing on eks

### DISCUSSION THREAD

**Discussion by KartikVelede** [gravitational/teleport#9966] [Q&A]
Hello everyone,
I am trying to install teleport on eks cluster with using helm chart and followed the below documentation
https://goteleport.com/docs/kubernetes-access/getting-started/cluster/
I have deployed teleport using below helm command
helm install teleport-cluster teleport/teleport-cluster --create-namespace --namespace=teleport-cluster \  --set clusterName=FQDN --set acme=true --set acmeEmail= email
After Executing the above command the deployment have been successfully completed and pod is running but if I am trying to access the application using DNS its not responding and I found the below error logs on pod

ERROR REPORT:
Original Error: *errors.errorString acme/autocert: missing server name
Stack Trace:
        /go/src/github.com/gravitational/teleport/lib/srv/alpnproxy/proxy.go:334 github.com/gravitational/teleport/lib/srv/alpnproxy.(*Proxy).handleConn
        /go/src/github.com/gravitational/teleport/lib/srv/alpnproxy/proxy.go:272 github.com/gravitational/teleport/lib/srv/alpnproxy.(*Proxy).Serve.func1
        /opt/go/src/runtime/asm_amd64.s:1581 runtime.goexit
User Message: acme/autocert: missing server name] alpnproxy/proxy.go:280
2022-01-26T17:02:05Z WARN [ALPN:PROX] Failed to handle client connection. error:[

Looking for some suggestions around this issue

**Comment by zen** [comment]
Have you created dedicated Route 53 zone for Teleport? Did you create the Load Balancer and pointed it to Teleport kubernetes service? Can you port forward to Teleport service without SSL?

**Comment by webvictim** [comment]
What's in your Teleport config file? `kubectl -n teleport-cluster get configmap/teleport-cluster -o yaml`
